contracts/minter-suite/Minters/MinterMerkle/V3Notes.md

```diff
@@ -0,0 +1,4 @@
+## TODOs for MinterMerkleV3
+
+- Emit event when assigning DELEGATION_REGISTRY_ADDRESS to enable off-chain indexing
+  - This was a constant in the original, original version of MerkleMinterV2, so indexing is not possible on that minter (a constant or call to the contract must be used on the frontend)
```

Comment:
Very minor typo nit ("original" twice) 😅

Reply:
ty! ended up removing this file since we decided to roll these changes into a new V3 minter 😄